lib/runtime/life/instance.go

```diff
@@ -116,7 +135,11 @@ func NewInstance(code []byte, cfg *Config) (runtime.Instance, error) {
 	}
 
 	ctx = runtimeCtx
-	inst.version, _ = inst.Version()
+	// todo fix this so that it doesn't panic if instVersion isn't in the wasm blob
+	//inst.version, err = inst.Version()
+	//if err != nil {
+	//	logger.Error("error checking instance version", "error", err)
+	//}
```

Comment:
I think you can remove this for now - maybe we can discuss potentially forking life to fix this ;)

Reply:
After updated instance to return error (instead of panic) on function not found, fixes this.  So, I've un-commented this, which also fixes the issue with the tests that I added skips to (since inst.version is no longer nil).